src/go/MONIT/alert.go

```diff
@@ -0,0 +1,658 @@
+package main
+
+// File       : alert.go
+// Author     : Rahul Indra <indrarahul2013 AT gmail dot com>
+// Created    : Tue, 9 June 2020 11:04:10 GMT
+// Description: CERN MONIT infrastructure Alert CLI Tool
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/http/httputil"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+	"text/tabwriter"
+	"time"
+)
+
+//-------VARIABLES-------
+//URL for AlertManager
+var cmsmonURL string
+
+//MAX timeStamp //Saturday, May 24, 3000 3:43:26 PM
+var maxtstmp int64 = 32516091806
+
+//alertname
+var name string
+
+//service name
+var service string
+
+//tag name
+var tag string
+
+//severity level
+var severity string
+
+//boolean for JSON output
+var jsonOutput *bool
+
+//boolean for detailed output for an alert
+var details *bool
+
+//Sort Label
+var sortLabel string
+
+//verbose defines verbosity level
+var verbose int
+
+//-------VARIABLES-------
+
+//-------MAPS-------
+
+//Map for storing filtered alerts
+var filteredAlerts map[string]int
+
+//Map for severityLevel (Sorting Helper)
+var severityLevel map[string]int
+
+//Map for storing Alert details against their name
+var alertDetails map[string]interface{}
+
+//-------MAPS-------
+
+//AlertManager API acceptable JSON Data for GGUS Data
+type amGGUSJSON struct {
```

Comment:
replace with amJSON struct which I provide in my review

Reply:
Fixed this issue.